packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -525,6 +525,33 @@ export const fastai = (model: ModelData): string[] => [
 learn = from_pretrained_fastai("${model.id}")`,
 ];
 
+export const sam2 = (model: ModelData): string[] => {
+	const image_predictor = `import torch
```

Comment:
Maybe add a comment (just as in transformers) at the top of each snippet. E.g.

```suggestion
	const image_predictor = `# Use SAM2 with images
import torch
```